### PR TITLE
feat: pod templates

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

* It adds 4 pod examples 

## Why is it important?

This pod templates can be used to launch Jenkins agents pods

### CODE REVIEW

File: resources/pods/dind-python.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          value: tcp://localhost:2375
+  volumes:
+    - name: docker-cache
+      emptyDir: {}
+  resources:
+    limits:
+      cpu: 2
+      memory: 8Gi
+    requests:
+      cpu: 2
```
Comment (kuisathaverat) [inline resources/pods/dind-python.yml:42]:
```suggestion
      cpu: 1
```

File: resources/pods/golang.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    command:
+      - sleep
+    args:
+      - infinity
+  resources:
+    limits:
+      cpu: 2
+      memory: 4Gi
+    requests:
+      cpu: 2
```
Comment (kuisathaverat) [inline resources/pods/golang.yml:18]:
```suggestion
      cpu: 1
```

File: resources/pods/python.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    command:
+      - sleep
+    args:
+      - infinity
+  resources:
+    limits:
+      cpu: 2
+      memory: 4Gi
+    requests:
+      cpu: 2
```
Comment (kuisathaverat) [inline resources/pods/python.yml:18]:
```suggestion
      cpu: 1
```

**Review by mdelapenya — APPROVED** [review]

**Review by v1v — APPROVED** [review]

**Comment by cachedout** [comment]
@kuisathaverat I assume this is part of the work outlined in https://github.com/elastic/observability-robots/issues/889 ?

**Review by cachedout — APPROVED** [review]

**Pull request merged**